.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
we shouldn't need to update this file, these things are handled by create-react-app. please remove.

Reply:
removed

Reply:
i'm not seeing this file as removed?

Reply:
its still there . file is not removed

Reply:
I can help you with this.